### PR TITLE
Replace Icons with ToggleSwitch for DisableProject column

### PR DESCRIPTION
**Pull request by novinxy**
This Pull Request replaces custom icons with toggleSwitch. 
Current icons are not clearly visible witch lot of projects, especially in Dark Theme.
ToggleSwitch is already used in Projects configuration page so it plays well with current UX.

### Testing done
Tested on local Jenkins instance
Before:
image

image
After:
image
image

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by novinxy** [comment]
Hi @fredg02 What do you think about this change?

**Comment by fredg02** [comment]
Thanks for your PR. I will take a look.

File: src/main/resources/jenkins/plugins/extracolumns/DisableProjectColumn/column.jelly [hunk 1]
```diff
@@ -53,4 +51,6 @@ THE SOFTWARE.
             <st:adjunct includes="jenkins.plugins.extracolumns.DisableProjectColumn.button"/>
        </j:if>
     </td> 
+
+    <st:adjunct includes="jenkins.plugins.extracolumns.DisableProjectColumn.disable-project"/>
```
Comment (fredg02) [inline src/main/resources/jenkins/plugins/extracolumns/DisableProjectColumn/column.jelly:55]: The following warning appears in the logs:
```
WARNING o.k.s.f.adjunct.AdjunctsInPage#findNeeded: No such adjunct found: 
jenkins.plugins.extracolumns.DisableProjectColumn.disable-project
org.kohsuke.stapler.framework.adjunct.NoSuchAdjunctException:
Neither jenkins.plugins.extracolumns.DisableProjectColumn.disable-project.css, .js, .html, nor .jelly were found
```
Not sure what the intention was, but I guess it works just fine without this adjunct.

Comment (novinxy) [reply src/main/resources/jenkins/plugins/extracolumns/DisableProjectColumn/column.jelly]: Thanks for pointing it, I completely overlooked it.
It was leftover of other changes and I removed that line.

In first version I also wanted to included change about CSP, but I was beaten to it by https://github.com/jenkinsci/extra-columns-plugin/pull/33

Comment (fredg02) [reply src/main/resources/jenkins/plugins/extracolumns/DisableProjectColumn/column.jelly]: No worries. Thanks again for the fix and your PR.

**Pull request merged**